cmd/crictl/container.go

```diff
@@ -530,6 +539,11 @@ var runContainerCommand = &cli.Command{
 		Name:    "runtime",
 		Aliases: []string{"r"},
 		Usage:   "Runtime handler to use. Available options are defined by the container runtime.",
+	}, &cli.Int64Flag{
```

Comment:
Let's use `cli.TimestampFlag` here.

Reply:
I am not so sure I like that. it requires users to do a diff of time at the command line, which seems more complicated than just specifying seconds.

Reply:
Ah I think I meant `DurationFlag`. Would this fit better? :innocent:

Reply:
ah yes that's better! updated as suggested

Reply:
Thank you! :pray: